v3/src/components/map/components/map-inspector.tsx

```diff
@@ -95,9 +95,6 @@ export const MapInspector = observer(function MapInspector({tile, show}: ITileIn
           <SaveImageMenuList tile={tile}/>
         </InspectorMenu>
         {renderPaletteIfAny()}
-      {showPalette === "measure" &&
-         <MapMeasurePalette tile={tile} setShowPalette={setShowPalette}
-                              panelRect={panelRect} buttonRect={buttonRect}/>}
```

Comment:
I removed this because it was causing two instances of the measure palette to be added to the DOM. I believe it must've been left in accidentally after `renderPaletteIfAny` was added or modified to handle adding the measure palette.